module/VuFind/src/VuFind/Navigation/AdminMenuFactory.php

```diff
@@ -71,6 +71,7 @@ public function __invoke(
             [
                 'AdminMenu.yaml',
                 $config['Overdrive']['showOverdriveAdminMenu'] ?? false,
+                ...$options ?? [],
```

Comment:
I think you need parentheses here:

```suggestion
                ...($options ?? []),
```